contracts/evm/xcall/test/CallService.t.sol

```diff
@@ -95,4 +108,80 @@ contract CallServiceTest is Test {
         callService.setProtocolFeeHandler(user);
     }
 
+    function testSendMessageSingleProtocol() public {
+        bytes memory data = bytes("test");
+        bytes memory rollbackData = bytes("");
+
+        vm.prank(address(dapp));
+        vm.expectEmit();
+        emit CallMessageSent(address(dapp), iconDapp, 1);
+
+        uint256 sn = callService.sendCallMessage{value: 0 ether}(iconDapp, data, rollbackData, _baseSource, _baseDestination);
+        assertEq(sn, 1);
+
+        Types.CSMessageRequest memory request = Types.CSMessageRequest(ethDappAddress, dstAccount, 1, false, data, new string[](0));
+        Types.CSMessage memory msg = Types.CSMessage(Types.CS_REQUEST, RLPEncodeStruct.encodeCSMessageRequest(request));
+
+        vm.mockCall(address(dapp), abi.encodeWithSelector(baseConnection.sendMessage.selector, netTo, xcallMulti, 0, msg), abi.encode(1));
```

Comment:
Should this not be vm.expectCall?
Also sn in this expect is supposed to be the sn that is returned when rollback is specified